### ISSUE TITLE
volar intercalated segment instability deformity

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10727]
**Preferred term label:**
volar intercalated segment instability deformity

**Synonyms**

**Definition (free text, please give PubMed ID)**
Lateral wrist radiograph showing volar flexion of the lunate with a scapholunate angle <30°

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Lunotriquetral (LT) ligament injury
Carpal instability nondissociative (CIND)

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001209
- label: Volar intercalated segment instability
- parents: [Abnormal carpal morphology (HP:0001191)](https://hpo.jax.org/app/browse/term/HP:0001191)
- pmid: PMID:32491661
- def: A type of carpal instability characterized by volar tilt of the lunate bone.
- comment: Lateral wrist radiograph showing volar flexion of the lunate with a scapholunate angle less than 30 degrees.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10727
- synonyms:
